lib/createTables.js

```diff
@@ -48,13 +49,13 @@ internals.waitTillActive = (model, callback) => {
 
         status = data.Table.TableStatus;
 
-        setTimeout(callback, 1000);
+        setTimeout(callback, options.pollingInterval || 1000);
       });
     },
   () => status !== 'ACTIVE',
   err => callback(err));
 };
 
 module.exports = (models, config, callback) => {
-  async.eachSeries(_.keys(models), (key, callback) => internals.createTable(models[key], config[key], callback), callback);
+  async.eachSeries(_.keys(models), (key, callback) => internals.createTable(models[key], config.$dynogels, config[key], callback), callback);
```

Comment:
From whoever makes the `createTables()` call.  For example, I'm using it like this in my service test setup procedure:

```js
Q.ninvoke(env.dynogels, 'createTables', { $dynogels: { pollingInterval: 10 } })
```